datalad/interface/base.py

```diff
@@ -100,6 +100,22 @@ def get_interface_groups(include_plugins=False):
     return grps
 
 
+def get_interface_name(intfspec, domain='python'):
```

Comment:
Doesn't `get_api_name` exist to handle "interface spec -> python api name" conversions?